app.py

```diff
@@ -33,6 +33,7 @@
 
 app = Flask(__name__)
 app.config.from_object(os.environ.get('APP_SETTINGS'))
+app.config['SECURITY_PASSWORD_SALT'] = '$2a$16$PnnIgfMwkOjGX4SkHqSOPO'
```

Comment:
Since this is meant to run locally, it would be ok to commit a random password salt. It may be helpful to leave a comment warning users to change this to a separate config if it is ever used in production.